CHANGELOG.md

```diff
@@ -47,6 +47,7 @@
   * `cortex_compactor_tenants_processing_failed`
 * [ENHANCEMENT] Added new experimental API endpoints: `POST /purger/delete_tenant` and `GET /purger/delete_tenant_status` for deleting all tenant data. Only works with blocks storage. Compactor removes blocks that belong to user marked for deletion. #3549 #3558
 * [ENHANCEMENT] Chunks storage: add option to use V2 signatures for S3 authentication. #3560
+* [ENHANCEMENT] Memberlist: add status page with available details about memberlist-based KV store and memberlist cluster. It's also possible to view KV values in Go struct or JSON format, or download for inspection. #3575
```

Comment:
**nit(optional):** It may be good to clarify the mechanism used to view/download.

```suggestion
* [ENHANCEMENT] Memberlist: add status page with available details about memberlist-based KV store and memberlist cluster. Additional query parameters can be added to view KV values in either Go struct or JSON format, or to download KV values for inspection. #3575
```

Reply:
I think people will mostly just click on the links in the page (see screenshot). I'm not sure this is worth having in changelog.